example/example_server/rebar.config

```diff
@@ -0,0 +1,7 @@
+{erl_opts, [debug_info]}.
+{deps, [{minirest, {git, "https://github.com/emqx/minirest", {tag, "1.0.0"}}}]}.
+
+{shell, [
+  % {config, "config/sys.config"},
```

Comment:
Forgotten comment?

Reply:
Is `comment` means some field of rebar.config?

Reply:
No, I was just wondering if the commented line is needed, or it can be removed

Reply:
OK, fixed